src/main/cms/cms_types.h

```diff
@@ -47,8 +47,6 @@ typedef enum
 
     // Debug aid
     OME_MENU,
-
-    OME_MAX = OME_MENU
```

Comment:
OME_MAX is used!
I realy like the idea from @AndersHoglund  here.
Just add OME_MAX back in and let it count up and change and then simply change the if condition
https://github.com/AlienWiiBF/betaflight/blob/61a3e91afc646090a2b8db445178af9f29fdb5c0/src/main/cms/cms.c#L441